### PR TITLE
Add updated README-QWIKLABS.md

### PR DESCRIPTION
**Pull request by mvoora**
/assign @chrislovecnm

### CODE REVIEW

File: README-QWIKLABS.md [hunk 1]
```diff
@@ -4,14 +4,11 @@
...
-	* [Run Demo in a SSH Terminal](#run-demo-in-a-ssh-terminal)
-	  * [Tools](#tools)
-	    * [Install Cloud SDK](#install-cloud-sdk)
-	    * [Install kubectl CLI](#install-kubectl-cli)
-	    * [Install Terraform](#install-terraform)
-	    * [Configure Authentication](#configure-authentication)
+* [Initial Setup](#initial-setup)
+   * [Configure gcloud](#configure-gcloud)
+* [Tools](#tools)
+   * [Install Apache Bench](#install-apache-bench)
```
Comment (bgeesaman) [inline README-QWIKLABS.md]: Are we missing the "Install Terraform" instructions section?

File: README-QWIKLABS.md [hunk 2]
```diff
@@ -36,23 +33,35 @@ pipeline. Provision a service account with all permissions needed to run this wh
...
+
+The specific versions used may not be absolutely required but if you run into issues this may help. Please note, Terraform has already been installed in Cloud console for you.
+
+#### Install Apache Bench
+
+For many users this won't be necessary as many operating systems have Apache
+Bench pre-installed. However it is contained within the `apache2-utils` package
+for Ubuntu/Debian users, and the `httpd-tools` package for CentOS/Redhat users.
+
+### Configure Authentication
```
Comment (bgeesaman) [inline README-QWIKLABS.md]: I think this section is missing a header in the TOC.

File: README-QWIKLABS.md [hunk 3]
```diff
@@ -36,23 +33,35 @@ pipeline. Provision a service account with all permissions needed to run this wh
...
+
+Use `gcloud auth login <your.account@example.com> --no-launch-browser` to get a link to log in your
```
Comment (bgeesaman) [inline README-QWIKLABS.md]: Can you tell me how a set of personal credentials are to be used inside qwicklabs?  If it's necessary, can we derive the account email from an env var somehow?  I can see folks authenticating with the incorrect account and not having feedback if that's correct or not as they proceed through the demo.

File: README-QWIKLABS.md [hunk 4]
```diff
@@ -36,23 +33,35 @@ pipeline. Provision a service account with all permissions needed to run this wh
...
 
 ```console
 gcloud init
 ```
 
+### Tools
+
+1. Apache Bench
+2. Terraform
+
```
Comment (homernajafi) [inline README-QWIKLABS.md]: Why do you mention terraFrom if it's already installed by CloudShell?

File: README-QWIKLABS.md [hunk 5]
```diff
@@ -36,23 +33,35 @@ pipeline. Provision a service account with all permissions needed to run this wh
...
+The specific versions used may not be absolutely required but if you run into issues this may help. Please note, Terraform has already been installed in Cloud console for you.
+
+#### Install Apache Bench
+
+For many users this won't be necessary as many operating systems have Apache
+Bench pre-installed. However it is contained within the `apache2-utils` package
+for Ubuntu/Debian users, and the `httpd-tools` package for CentOS/Redhat users.
+
```
Comment (homernajafi) [inline README-QWIKLABS.md]: Just say run:   sudo apt-get update && sudo apt-get install apache2-utils

File: README-QWIKLABS.md [hunk 6]
```diff
@@ -36,23 +33,35 @@ pipeline. Provision a service account with all permissions needed to run this wh
...
+### Configure Authentication
+
+Use `gcloud auth login <your.account@example.com> --no-launch-browser` to get a link to log in your
+gcloud cli to your personal account.
```
Comment (homernajafi) [inline README-QWIKLABS.md]: I don't think we need gcloud auth.   CloudShell will log you in.

**Pull request merged**